index.html

```diff
@@ -1724,6 +1724,80 @@ <h4>CredentialRegistry</h4>
         </pre>
       </section>
 
+      <section>
+        <h4>OpenID4 Verifiable Credential Issuance</h4>
+        <p>The <code>OID4VCI</code> service allows publication of the <a href="https://openid.net/openid4vc/">OpenID for Verifiable Credential Issuance (OID4VCI)</a> domain, where the <code>.well-known</code> files (<code>oauth-authorization-server</code> & <code>openid-credential-issuer</code>) can be found. These files MUST contain the endpoints and meta data for <a href="https://openid.net/specs/openid-4-verifiable-credential-issuance-1_0.html">OpenID for Verifiable Credential Issuance (OpenID4VCI)</a>.</p>
+        <p>The service endpoint `id` MUST be the Credential Issuer Identifier to get the Credential Issuer Metadata as described in Section <a href="https://openid.net/specs/openid-4-verifiable-credential-issuance-1_0.html#credential-issuer-metadata">Credential Issuer Metadata</a> of <a href="https://openid.net/specs/openid-4-verifiable-credential-issuance-1_0.html">OpenIDVCI</a>.</p>
+        <table class="simple" style="width: 100%;">
+          <thead>
+            <tr>
+              <th>Normative Definition</th>
+              <th>JSON-LD</th>
+            </tr>
+          </thead>
+          <tbody>
+            <tr>
+              <td>
+                <a href="https://openid.net/specs/openid-4-verifiable-credential-issuance-1_0.html">OpenID4VCI</a>
+              </td>
+              <td>
+                <a href="https://ssi.eecc.de/api/registry/context/openid4vc">OpenID4VCI</a>
+              </td>
+            </tr>
+          </tbody>
+        </table>
+
+        <pre class="example" title="Example of service and serviceEndpoint properties">
+{
+  ...
+  "service": [
+    {
+      "id": "did:example:123#oid4vci",
+      "type": "OID4VCI",
+      "serviceEndpoint": "https://did.example.com/"
+    }
+  ]
+}
+        </pre>
+      </section>
+
+      <section>
+        <h4>OpenID4 Verifiable Presentation</h4>
+        <p>The <code>OID4VP</code> service allows publication of the <a href="https://openid.net/openid4vc/">OpenID for Verifiable Presentation (OID4VP)</a> authorization endpoint, which is the entry point for the <a href="https://openid.net/specs/openid-4-verifiable-presentations-1_0.html">presentation flow</a>.</p>
+        <p>The service endpoint `id` MUST be the authorization request endpoint as described in section <a href="https://openid.net/specs/openid-4-verifiable-presentations-1_0.html#name-authorization-request">Authorization Request</a>.</p>
```

Comment:
```suggestion
        <p>The <code>OID4VP</code> service allows publication of a credential wallet that conforms with the <a href="https://openid.net/specs/openid-4-verifiable-presentations-1_0.html">OpenID for Verifiable Presentation (OID4VP)</a> specification.</p>
        <p>The service endpoint `id` MUST be the Wallet (OAuth2) Issuer Identifier to obtain Wallet Metadata to invoke the wallet as described in section <a href="https://openid.net/specs/openid-4-verifiable-presentations-1_0.html#name-wallet-invocation">Wallet Invocation</a>.</p>
```

Reply:
What I don't like with this approach is that there can be potential privacy implications if wallet providers provide URLs per user (which is usually not the case). This can be fixed if we would include the Wallet Metadata in the service endpoint directly.